qt/scientific_interfaces/EnggDiffraction/EnggDiffMultiRunFittingQtWidget.cpp

```diff
@@ -0,0 +1,108 @@
+#include "EnggDiffMultiRunFittingQtWidget.h"
+
+#include "MantidKernel/make_unique.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+EnggDiffMultiRunFittingQtWidget::EnggDiffMultiRunFittingQtWidget(
+    boost::shared_ptr<IEnggDiffMultiRunFittingWidgetPresenter> presenter,
+    boost::shared_ptr<IEnggDiffractionUserMsg> messageProvider)
+    : m_presenter(presenter), m_userMessageProvider(messageProvider) {
+  setupUI();
+
+  m_zoomTool = Mantid::Kernel::make_unique<QwtPlotZoomer>(
```

Comment:
Yes. `m_zoomTool` can't be initialised before `m_ui.setup()` is called (before that, all `m_ui`'s members are null pointers), so it can't go in the init list. Copy constructors are disabled for `QObject`s (of which `m_zoomTool` is one), so we can't initialise `m_zoomTool` directly. It's this or `boost::optional`, and I think this is clearer